src/routes/mods.rs

```diff
@@ -0,0 +1,13 @@
+use actix_web::{get, web, HttpResponse};
+use crate::models::mods::SearchRequest;
+use crate::search::search_for_mod;
+
+#[get("api/v1/mods")]
+pub fn mod_search(web::Query(info): web::Query<SearchRequest>) -> HttpResponse {
+    //TODO: Fix this line with anyhow
+    let body = serde_json::to_string(&search_for_mod(&info).unwrap()).unwrap();
```

Comment:
Unwraps should be removed before merge

Reply:
I have to add anyhow to everything. I think this should be fine for now

Reply:
True, either way I don't think the second unwrap can fail. The first one can panic if invalid filters are specified